solang-server/src/main.rs

```diff
@@ -0,0 +1,18 @@
+use tower_lsp::{LspService, Server};
+
+#[allow(non_snake_case)]
+mod ServerUtils;
```

Comment:
Can we rename `ServerUtils` into something rust accepts without `#[allow(non_snake_case)]` please

Reply:
done